tests/foreman/sys/test_rename.py

```diff
@@ -0,0 +1,205 @@
+# -*- encoding: utf-8 -*-
+"""Test class for ``katello-change-hostname``
+
+@Requirement: katello-change-hostname
+
+@CaseAutomation: notautomated
+
+@CaseLevel: System
+
+@CaseComponent: katello-change-hostname
+
+@TestType: Functional
+
+@CaseImportance: High
+
+@Upstream: No
+"""
+from fauxfactory import gen_string
+from robottelo.config import settings
+from robottelo.decorators import (
+        destructive,
+        run_in_one_thread,
+        stubbed,
+)
+from robottelo.ssh import get_connection
+from robottelo.test import TestCase
+
+BCK_MSG = "Hostname change complete!"
+BAD_HN_MSG = "{0} is not a valid fully qualified domain name, please \
+use a valid FQDN and try again."
+NO_CREDS_MSG = "Username and/or Password options are missing!"
+BAD_CREDS_MSG = "Invalid username or password"
+
+
+@destructive
+class RenameHostTestCase(TestCase):
+    """Implements ``katello-change-hostname`` tests"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Get hostname and credentials"""
+        super(RenameHostTestCase, cls).setUpClass()
+        cls.hostname = settings.server.hostname
+        cls.username = settings.server.admin_username
+        cls.password = settings.server.admin_password
+
+    @stubbed
```

Comment:
why stubbed ?

Reply:
Hello, it's the same story as in https://github.com/SatelliteQE/robottelo/pull/5051, the code down there is just a note for future developer, the point here is really to get those negative tests in